frontend/jest.config.js

```diff
@@ -12,7 +12,7 @@ module.exports = {
     "^~~/(.*)$": "<rootDir>/$1",
     "^vue$": "vue/dist/vue.common.js",
     "(.*svg)(\\?inline)$": "<rootDir>/test/unit/test-utils/svgTransform.js",
-    axios: "axios/dist/node/axios.cjs",
+    "^axios$": "axios/dist/node/axios.cjs",
```

Comment:
Oh, this is so....unexpected! I spent a lot of time trying to figure out the fix, but I would never think of looking into this import! Thank you for finding the fix, @sarayourfriend!

Reply:
It all came down to attaching a debugger and noticing that `new AxiosMockAdapter` raised an error in the debug console and then inspecting the imported object and realising "oh... that's Axios!" and then I noticed that the keys here are regexes. I actually did a :facepalm: when I realised what a mistake I'd made :sweat_smile: